languages/csharp/BitwardenSafeHandle.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Runtime.InteropServices;
+
+namespace Bit.Sdk;
+
+internal class BitwardenSafeHandle : SafeHandle
```

Comment:
You can extend `SafeHandleZeroOrMinusOneIsInvalid` instead which lets you remove the `IsInvalid`.